app/src/main/res/values/strings.xml

```diff
@@ -869,9 +869,12 @@ The new line here must be kept as the second half of the string is clickable for
     <!-- CFR for Toolbar Shield Icon (Banner Info Message) -->
     <string name="cfr_for_toolbar_shield_icon">Looks like this site was trying to track you. Click the shield for site settings and tracker blocking stats.</string>
 
-    <!-- Tool Tip for: Privacy and Security Settings  -->
+    <!-- Tool Tip for: Privacy and Security Settings Screen  task : https://github.com/mozilla-mobile/focus-android/issues/6211 -->
+    <!-- Tool Tip Title -->
     <string name="tool_tip_title">You’re protected!</string>
+    <!-- Tool Tip Message -->
     <string name="tool_tip_message">Something about optimizing their protection, but also educating on what is being blocked, etc.</string>
```

Comment:
confirmed with @amedyne that this is a placeholder and should not appear as a string.
CC @iorgamgabriel

Reply:
@Delphine @amedyne I can't delete the string from resources because will appear a cfr with no message . I can put translatable=false at the string or @jeffreygee can provide the correct string . Thank you !

Reply:
Oh.. I see now that this is for this space: https://github.com/mozilla-mobile/focus-android/issues/6211. @jeffreygee  FYI. We need the actual strings for this. @iorgamgabriel let's mark this as translatable= false for now until we get the correct string from Jeff. Thanks!

Reply:
Checked figma, updated string is: 
<img width="351" alt="Screen Shot 2022-02-17 at 9 39 53 AM" src="https://user-images.githubusercontent.com/68711330/154504634-9965baae-489a-446a-9059-fb53d5d04f0c.png">
@iorgamgabriel FYI.

Reply:
@Delphine I made the changes .